cms/templates/accessibility.html

```diff
@@ -0,0 +1,27 @@
+<%page expression_filter="h"/>
+>
+<%inherit file="base.html" />
+<%def name="online_help_token()"><% return "accessibility" %></%def>
+<%!
+  from django.core.urlresolvers import reverse
+  from django.utils.translation import ugettext as _
+  from openedx.core.djangolib.markup import HTML, Text
+  from openedx.core.djangolib.js_utils import js_escaped_string, dump_js_escaped_json
+%>
+<%block name="title">${_("Accessibility")}</%block>
```

Comment:
Oops, I just found out that the title (via Mark) should be "Studio Accessibility Policy".

Reply:
Should the footer link be the same?

Reply:
Nope, the footer should be as is. Thanks for checking!